libs/styles/iconography.css

```
@@ -101,6 +98,9 @@ dealing w/ groups of media and associated text
   gap: 20px;
 }
 
+.right:not(.tabs.right) .lockup-area { justify-content: flex-end; }
+.center:not(.tabs.center):not(.section.center) .lockup-area { justify-content: center; }
```

Wouldn't this be a shorter equivalent?
```suggestion
.right:not(.tabs) .lockup-area { justify-content: flex-end; }
.center:not(.tabs, .section) .lockup-area { justify-content: center; }
```